src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$color-accent: #060b35;
+$color-secondary: #616070;
+$color-blue: #00acdc;
+$color-white: #fff;
+$color-concrete: #f3f3f3;
+$color-black-shadow: rgba(0, 0, 0, 0.05);
+
+$card-width: 200px;
+$star-size: 16px;
+$cards-gap-right: 48px;
+$cards-gap-bottom: 46px;
+$content-max-width: $card-width * 4 + $cards-gap-right * 3;
```

Comment:
Better to add variables for amount of cards and amount of spaces (maybe you will need to put only 3 cards in a row 😄 )